server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -393,7 +400,7 @@ public short partitionRecordVersion() {
     }
 
     public short fetchRequestVersion() {
-        if (this.isAtLeast(IBP_3_7_IV0)) {
+        if (this.isAtLeast(IBP_3_7_IV4)) {
```

Comment:
Noting that we do not have a test for this.  In the interest of time, let's add a test for this in a follow-on PR rather than here unless we find other issues.